### PR TITLE
Add root home page with controller

### PR DESCRIPTION
**Pull request by jestradadeveloper**
# Description
Add a root route to a home controller and index action with a view to print a welcome message

 Examples:
 - What?: add a root page for the landing home creation
 - Why?:  in order to have a home root in a heroku deployment

 # Testing
run the rspec command

  ## Screenshots
Captura de Pantalla 2021-11-11 a la(s) 18 55 33

### CODE REVIEW

**Review by JuanequeX — DISMISSED** [review]
LGTM

File: spec/requests/home_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe "Homes", type: :request do
+  describe "GET /" do
+    it "returns http success" do
+      get "/"
```
Comment (YilKanda) [inline spec/requests/home_spec.rb:6]: You can also put the root path.

```suggestion
      get root_url
```

**Review by YilKanda — DISMISSED** [review]
LGTM!

File: spec/requests/home_spec.rb (hunk 1 above)
Comment (jestradadeveloper) [reply spec/requests/home_spec.rb]: I already made this change.

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by adrianMRJ — APPROVED** [review]
LGTM

**Pull request merged**